apps/condo/domains/ticket/access/TicketChange.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Generated by `createschema ticket.TicketChange 'ticket:Relationship:Ticket:CASCADE;'`
+ */
+
+async function canReadTicketChanges ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    return {
+        // TODO(codegen): write canReadTicketChanges logic!
+    }
+}
+
+async function canManageTicketChanges ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+    if (operation === 'create') {
+        // TODO(codegen): write canManageTicketChanges create logic!
+        return true
```

Comment:
! why you need to create it by any user?

Reply:
I did't supposed to.
It's an untouched result of createschema, as I mentioned in `9df8a3a2`.
Of course, I will remove much of scaffolding later.